### DISCUSSION TITLE
O大，你好，求助B860AV3.1-M2   905L3B

### DISCUSSION THREAD

**Discussion by Ermi999** [ophub/amlogic-s9xxx-armbian#2214] [Amlogic]
![IMG_20240407_020654.jpg](https://github.com/ophub/amlogic-s9xxx-armbian/assets/159701937/d12cee5f-b5d7-4058-9e77-a18b450ef541)

这个我试了很多包，就是刷不过去。root底包启动外部U盘引导，然后就卡死不动了，插上网线也没有他获取IP的痕迹。
多个包都是。求助。

**Comment by mlinuxs** [comment]
我的跟你一样的型号，一开始我也是刷了好多个包都无法获取ip，后来我直接用TTL直接看启动，实际上已经启动了，只是无法识别有线和无线的网卡，所以无法获取IP地址。无奈只好插了一个以前的360USB无线网卡才能获取IP，SSH进入系统。

**Reply by Ermi999** [reply]
ssh连上后，网卡驱动怎么搞通的。

**Reply by mlinuxs** [reply]
目前盒子自带的有线/无线均无法正常使用，只能用USB网卡玩玩。
